app/src/main/AndroidManifest.xml

```diff
@@ -38,5 +38,21 @@
             android:configChanges="keyboardHidden|orientation"
             android:screenOrientation="sensorLandscape"
             android:theme="@style/Theme.AppCompat.FullScreen" />
+
+        <activity
+            android:name="com.libopenmw.openmw.FileChooser"
+            android:label="Choose File"
+            android:screenOrientation="sensorLandscape">
+
+            <intent-filter>
+                <action android:name="com.libopenmw.openmw.FileChooser" />
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
+        </activity>
+        
+        <meta-data
```

Comment:
The apikey should be removed, it was a mistake/laziness on my part.

Reply:
will this break CI?

Reply:
I don't know, I don't use travis for this repo. I was actually going to remove everything travis-related, but if it makes your life easier I can keep it around.

Reply:
For me removing the apikey broke the CI. You can see this here: https://travis-ci.org/OpenMW/android-port/builds/524194894#L1569

Reply:
In this case you should figure how to fix the CI (maybe again with a build flavor). Keeping a fabric key hardcoded isn't a good idea especially for an open source project (consider forks/different package names).